.github/workflows/test-bundler.yml

```diff
@@ -96,11 +96,6 @@ jobs:
             override: true
             components: rustfmt, clippy
 
-      - name: install webkit2gtk (ubuntu only)
```

Comment:
We need webkit.

Reply:
This step is never run since `if` condition is always false. See Actions result.